Gemfile

```diff
@@ -5,7 +5,7 @@ ruby '~> 2.6.5'
 
 gem 'rails', '~> 6.0.0'
 
-gem 'ahoy_matey', '~> 2.2', '>= 2.2.1'
+gem 'ahoy_matey', '~> 3.0'
```

Comment:
we rely heavily on the format JSON of ahoy in our logs and parsing, have we spot-checked this to make sure the output to events.log is still good?

Reply:
I took a peek and the format seems the same. The bigger changes in the upgrade were around bot detection logic and dropping support for older versions of Rails: https://github.com/ankane/ahoy/blob/master/CHANGELOG.md#300-2019-05-29